advertising/transaction-message/reference.md

```diff
@@ -147,14 +153,22 @@ Defines a container object that contains details about occupancy.
 |NumAdults|Required.<br />Data type is integer.<br /><br />The maximum number of adults the room is intended to accommodate.|None
 |Children|Optional.<br /><br />A container object the identifies the ages of children the room is intended to accommodate.|[Children Type](#children-type)
```

Comment:
Should we add a separate NumChildren Type or are we relying on the number of entries in the vector?

Reply:
Sticking with G! let's use # of entries in the vector